Add javacOpt using directive (#1438)
VirtusLab/scala-cli: website/docs/commands/compile.md

Note that you should favor the [`run`](./run.md) command to run your code, rather than running `java -cp`.
The class path obtained this way is only meant for scenarios where `scala-cli` doesn't offer a more convenient option.

### JVM options

> `--javac-opt` lets you add javac options which will be passed during compiling things. 

Reviewer: ```suggestion
`--javac-opt` lets you add `javac` options which will be passed when compiling sources. 
```
